robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,112 @@
+from itertools import tee
+
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+def eol():
+    return Token(Token.EOL)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120):
+        self.line_length = line_length
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(token.end_col_offset <= self.line_length for token in node.tokens[::-1]):
+            return node
+        if not node.get_tokens(Token.ARGUMENT):  # return if there are no arguments - nothing to split
+            return node
+        return self.split_keyword_call(node)
+
+    def split_keyword_call(self, node):
+        is_single_line = node.lineno == node.end_lineno
+
+        indent = node.tokens[0]
+        assignment = list(self.insert_seperator(node.get_tokens(Token.ASSIGN)))
+        keyword = node.get_token(Token.KEYWORD)
+
+        if is_single_line:
+            comment = node.get_tokens(Token.COMMENT)
+            comment_line = [indent] + comment + [eol()] if comment else []
```

Comment:
It's not going to preserve comments with arguments?
```
Keyword
...    ${arg}   # i am comment to arg
...    ${multiple arguments in one line}    ${multiple arguments in one line}    ${multiple arguments in one line}
...    ${arg2}  # i am comment to this
```
With this code it will yield to (or similiar):
```
# i am comment to arg  i am comment to this
Keyword (...and splitted args...)
```

Reply:
The example you've shown is not a single line, it will be handled by `else:` branch below.